package.json

```diff
@@ -20,6 +20,7 @@
     "build": "yarn clean && node . build",
     "serve": "yarn build && cd ./public && ws",
     "develop": "yarn clean && node . develop",
+    "eject":  "node . eject --all",
```

Comment:
I don't think we'll plan on ejecting, or at least not ever to the point we would need a dedicated script for a one time operation 😃

Reply:
for testing purposes, I had this of course.